src/util/util_env.cpp

```diff
@@ -130,62 +130,52 @@ namespace dxvk::env {
-        return getExecutableName() == std::string("MonsterHunterWorld.exe");
-    }
-
-    bool isWarThunder() {
-        return getExecutableName() == std::string("aces.exe");
-    }
-
     bool needsPascalSpoofing(NV_GPU_ARCHITECTURE_ID architectureId) {
-        if (architectureId >= NV_GPU_ARCHITECTURE_TU100 && isMonsterHunterWorld()) {
-            log::info("Spoofing Pascal for Turing and later due to detecting MonsterHunterWorld.exe");
+        if (architectureId >= NV_GPU_ARCHITECTURE_TU100 && getExecutableName() == std::string("MonsterHunterWorld.exe")) {
+            log::info("Spoofing Pascal for Turing and later due to detecting MonsterHunterWorld.exe (Monster Hunter World)");
             return true;
         }
 
-        if (architectureId >= NV_GPU_ARCHITECTURE_TU100 && isWarThunder()) {
+        if (architectureId >= NV_GPU_ARCHITECTURE_TU100 && getExecutableName() == std::string("aces.exe")) {
             log::info("Spoofing Pascal for Turing and later due to detecting aces.exe (War Thunder)");
             return true;
         }
 
         return false;
     }
 
-    bool isTheLastOfUsPartOne() {
-        return getExecutableName() == std::string("tlou-i.exe")
-            || getExecutableName() == std::string("tlou-i-l.exe");
-    }
-
-    bool isTheLastOfUsPartTwo() {
-        return getExecutableName() == std::string("tlou-ii.exe")
-            || getExecutableName() == std::string("tlou-ii-l.exe");
-    }
-
-    bool isXDefiant() {
-        return getExecutableName() == std::string("XDefiant.exe")
-            || getExecutableName() == std::string("XDefiant_BE.exe");
-    }
-
     bool needsSucceededGpuQuery() {
-        if (isTheLastOfUsPartOne()) {
-            log::info("Faking GPU query success due to detecting tlou-i.exe/tlou-i-l.exe");
+        if (getExecutableName() == std::string("tlou-i.exe") || getExecutableName() == std::string("tlou-i-l.exe")) {
+            log::info("Faking GPU query success due to detecting tlou-i.exe/tlou-i-l.exe (The Last of Us Part I)");
             return true;
         }
 
-        if (isTheLastOfUsPartTwo()) {
-            log::info("Faking GPU query success due to detecting tlou-ii.exe/tlou-ii-l.exe");
+        if (getExecutableName() == std::string("tlou-ii.exe") || getExecutableName() == std::string("tlou-ii-l.exe")) {
+            log::info("Faking GPU query success due to detecting tlou-ii.exe/tlou-ii-l.exe (The Last of Us Part II)");
             return true;
         }
 
-        if (isXDefiant()) {
+        if (getExecutableName() == std::string("XDefiant.exe") || getExecutableName() == std::string("XDefiant_BE.exe")) {
             log::info("Faking GPU query success due to detecting XDefiant.exe/XDefiant_BE.exe (XDefiant)");
             return true;
         }
 
         return false;
     }
 
+    bool needsUnsupportedLowLatencyDevice() {
+        static bool alreadyLogged = false;
+
+        if (getExecutableName() == std::string("Stormgate-Win64-Shipping.exe")) {
```

Comment:
Since this is on a hot path, I'd prefer we evaluated it only once and cached the result. Maybe something like

```suggestion
        static bool isStormgate = getExecutableName() == std::string("Stormgate-Win64-Shipping.exe");

        if (isStormgate) {
```

would work?

Reply:
I went for the `static bool`, looks fine on paper.